app/v3/controllers/RetrieveSelfEmploymentBsasController.scala

```diff
@@ -65,9 +65,7 @@ class RetrieveSelfEmploymentBsasController @Inject()(
           parsedRequest <- EitherT.fromEither[Future](requestParser.parseRequest(rawData))
           response      <- EitherT(service.retrieveSelfEmploymentBsas(parsedRequest))
           hateoasResponse <- EitherT.fromEither[Future](
-            hateoasFactory
-              .wrap(response.responseData, RetrieveSelfAssessmentBsasHateoasData(nino, response.responseData.metadata.calculationId))
-              .asRight[ErrorWrapper])
+            hateoasFactory.wrap(response.responseData, RetrieveSelfAssessmentBsasHateoasData(nino, response.responseData.metadata.calculationId)).asRight[ErrorWrapper])
         } yield {
           logger.info(
             s"[${endpointLogContext.controllerName}][${endpointLogContext.endpointName}] - " +
```

Comment:
From below, please remove:
```
.as(MimeTypes.JSON)
```
And from `v3.controllers.BaseController` (and `BaseControllerSpec`) please remove the line:
```
"Content-Type" -> "application/json"
```
which will cause Play warnings to be logged.